### DISCUSSION TITLE
Delayed time remaining attribute?

### DISCUSSION THREAD

**Discussion by mash2k3** [sebr/bhyve-home-assistant#199] [Ideas]
I noticed on app and portal it displays how much time is left for delay, wondering if that can be added as an attribute?
Probably possible to create a template using delay and started at attribute but thought would be easier from api.

**Comment by sebr** [comment]
This isn't something I plan on adding as "time remaining" is relative and would rely updating the attribute every second. For users who wish to use this a better approach would be a dynamic lovelace card which has a countdown timer or a template sensor if you needed to use this elsewhere in your setup.
